src/DrawingModule.cpp

```diff
@@ -29,9 +29,6 @@ void DrawingModule::update() {
 	monsters->ordena();	
 	obstacles->ordena();
 
-	// Limpa a tela
-	window->clear();
```

Comment:
É realmente necessário tirar clear do modulo?

Reply:
WIP Significa Work In Progress. Este pull request não deve ser revisado ainda

Reply:
Tudo bem :+1: